docs/v3/core/v3.0.rst

```diff
@@ -597,30 +597,57 @@ mandatory names:
     Provides an element value to use for uninitialised portions of the
     Zarr array.
 
-    If the data type of the Zarr array is Boolean then the value must
-    be the literal ``false`` or ``true``. If the data type is one of
-    the integer data types defined in this specification, then the
-    value must be a number with no fraction or exponent part and must
-    be within the range of the data type.
+    The permitted values depend on the data type:
 
-    For any data type, the ``fill_value`` is required. The literal
-    ``null`` is not permitted. The fill value needs to be defined
-    so that the data is independent of implementation details. Internally
-    implementations may provide a default ``fill_value``, but that must
-    be converted to a fixed value in the stored metadata.
```

Comment:
I think it might be useful to keep this sentence, to distinguish between metadata and library API level.

Reply:
Done.